### PR TITLE
removed duplicated code by using csproj with targetframeworks

### PR DESCRIPTION
**Pull request by JanEggers**

### CODE REVIEW

File: Frameworks/MQTTnet.NetStandard/MQTTnet.Netstandard.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (JanEggers) [inline Frameworks/MQTTnet.NetStandard/MQTTnet.Netstandard.csproj]: this was what i meant with .net.sdk ^^

**Comment by chkr1011** [comment]
Did you test the Build script? I usually use this only when deploying a new version. I assume the line for the NetFramework must be changed to build the other project using a different config.

**Comment by JanEggers** [comment]
i updated the buildscript so the .net standard is build for both platforms, i didnt get msbuild to work and i dont know your setup as msbuild is not found on my machine because it is not on PATH

**Comment by chkr1011** [comment]
Yes I usually use the VS2017 developer command line. If your approach works it is fine for me. But still one question: Isn't it required that the project is compiled two times with different configruations? Because of the #ifdef?

**Comment by JanEggers** [comment]
nope dotnet always builds all frameworks used in csproj except you specifiy a framework with -f
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build?tabs=netcore2x

**Pull request merged**